### PR TITLE
Add a couchbase cluster check

### PR DESCRIPTION
**Pull request by obazoud**
Add a replication number check

### CODE REVIEW

File: bin/check-couchbase-bucket-replica.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: rest-client
+#   Gem: json
+#
+# USAGE:
+#
```
Comment (eheydrick) [inline bin/check-couchbase-bucket-replica.rb:22]: would be nice to have usage.

File: bin/check-couchbase-cluster.rb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: rest-client
+#   Gem: json
+#
+# USAGE:
+#
```
Comment (eheydrick) [inline bin/check-couchbase-cluster.rb:21]: would be nice to have usage.

**Comment by eheydrick** [comment]
Thanks for the new plugins! Looks good, left a few comments. Only other thing needed is an entry in the CHANGELOG under Unreleased.

**Comment by obazoud** [comment]
@eheydrick done

**Pull request merged**